src/themes/default/modules/dropdown.variables

```diff
@@ -252,12 +253,14 @@
 @multipleSelectionVerticalPadding: (@searchSelectionVerticalPadding * (1/3));
 @multipleSelectionLeftPadding: @relative5px;
 @multipleSelectionRightPadding: @selectionIconDistance;
-@multipleSelectionPadding: @multipleSelectionVerticalPadding @multipleSelectionRightPadding @multipleSelectionVerticalPadding @multipleSelectionLeftPadding;
+@multipleSelectionPadding: @multipleSelectionVerticalPadding @multipleSelectionRightPadding
+    @multipleSelectionVerticalPadding @multipleSelectionLeftPadding;
 
 /* Child Elements */
 @multipleSelectionChildVerticalMargin: (@searchSelectionVerticalPadding * (2/3));
 @multipleSelectionChildLeftMargin: (@inputHorizontalPadding - @multipleSelectionLeftPadding);
-@multipleSelectionChildMargin: @multipleSelectionChildVerticalMargin 0 @multipleSelectionChildVerticalMargin @multipleSelectionChildLeftMargin;
+@multipleSelectionChildMargin: @multipleSelectionChildVerticalMargin 0 @multipleSelectionChildVerticalMargin
```

Comment:
Same here, putting only the last margin into a separate line makes no sense to me